client.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/anchore/go-logger"
```

Comment:
nit: it would be nice if we were using `gosimports` here

Reply:
I agree, I can follow up in another PR to get stereoscope up to date with some of the other repos